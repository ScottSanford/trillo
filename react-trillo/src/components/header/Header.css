
.header {
    align-items: center;
    background-color: var(--color-white);
    border-bottom: var(--line);
    display: flex;
    font-size: 1.4rem;
    height: 7rem;
    justify-content: space-between; 
}

@media only screen and (max-width: 31.25em) {
	.header {
		align-content: space-around; 
		flex-wrap: wrap;
		height: 11rem;
		margin-bottom: 1.5rem;
	}
}

/* LOGO */
.logo {
    height: 3.25rem;
    margin-left: 2rem;
}

.u--margin-bottom-small {
    margin-bottom: 2rem;
}

.btn-inline {
    background-color: transparent;
    border: none;
    border-bottom: 1px solid currentColor; /* currentColor allows the hover color to change as well */
    color: var(--color-primary);
    cursor: pointer;
    display: inline-block;
    font-size: inherit;
    padding-bottom: 2px;
    transition: all .2s;
	
}

.btn-inline span {
	margin-left: .3rem;
	transition: margin-left .2s;
}

.btn-inline:hover {
	color: var(--color-grey-dark-1);
}

.btn-inline:hover span {
	margin-left: 8px;
}

.btn-inline:focus {
	animation: pulsate 1s infinite;
	outline: none;
}

@keyframes pulsate {
    0% {
        box-shadow: none;
        transform: scale(1);
    }

    50% {
        box-shadow: 0 1rem 4rem rgba(0,0,0, .25);
        transform: scale(1.05);
    }

    100% {
        box-shadow: none;
        transform: scale(1);
    }
}
