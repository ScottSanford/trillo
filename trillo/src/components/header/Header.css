
.header {
    align-items: center;
    background-color: var(--color-white);
    border-bottom: var(--line);
    display: flex;
    font-size: 1.4rem;
    height: 7rem;
    justify-content: space-between; 
}

@media only screen and (max-width: 31.25em) {
	.header {
		align-content: space-around; 
		flex-wrap: wrap;
		height: 11rem;
		margin-bottom: 1.5rem;
	}
}

/* LOGO */
.logo {
    height: 3.25rem;
    margin-left: 2rem;
}

.u--margin-bottom-small {
    margin-bottom: 2rem;
}
