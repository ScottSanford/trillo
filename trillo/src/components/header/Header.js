import React from 'react'
import './Header.css'
import SearchInput from './search-input/search-input'
import Navigation from './navigation/navigation'

export default function Header() {
  return (
    <div className="header">
      <img src="../../img/logo.png" alt="Trillo Logo" className="logo" />
      <SearchInput />
      <Navigation />
    </div>
  )
}