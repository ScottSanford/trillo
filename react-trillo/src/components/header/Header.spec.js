import React from 'react'
import { shallow } from 'enzyme'
import Header from './Header'

describe('Header Component', () => {

	it('should match snapshot', () => {
		const wrapper = shallow(<Header />)
		expect(wrapper).toMatchSnapshot()
	})
})