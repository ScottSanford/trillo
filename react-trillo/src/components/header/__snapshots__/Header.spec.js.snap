// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should match snapshot 1`] = `
<div
  className="header"
>
  <img
    alt="Trillo Logo"
    className="logo"
    src="../../img/logo.png"
  />
  <SearchInput />
  <Navigation />
</div>
`;
